system/jhtree/jhtree.cpp

```diff
@@ -1136,14 +1160,12 @@ CJHTreeNode *CDiskKeyIndex::loadNode(cycle_t * fetchCycles, CJHTreeNode * optNod
     }
     if (fetchCycles)
         *fetchCycles = fetchTimer.elapsedCycles();
-    if (optNode)
-        return CKeyIndex::loadNode(optNode, nodeData, pos, true);
-    return CKeyIndex::loadNode(nodeData, pos, true);
+    return CKeyIndex::_loadNode(nodeData, pos, true);
 }
 
-CJHTreeNode *CKeyIndex::createNode(NodeType type)
+CJHTreeNode *CKeyIndex::_createNode(const NodeHdr &nodeHdr) const
```

Comment:
Could rename this back to createNode(), or is it deliberately kept with an _?

Reply:
I used the _ to indicate internal... it's a bit inconsistent though, and I don't have strong feelings